forms/admission/person_merge_proposal_form.py

```diff
@@ -75,3 +76,8 @@ class Meta:
             'unknown_birth_date',
             'already_registered',
         ]
+        force_translations = [
+            _('Civil state'),
+            _('Middle name'),
+            _('Country of citizenship name')
```

Comment:
pourquoi pas Country of citizenship qui est déjà traduit?
Idem pour Civil status ?

Reply:
On pourrait envisager une approche linguistique axée sur la préservation de la nuance sémantique en maintenant les termes 'Country of Citizenship' et 'Civil Status' dans leur forme anglaise, afin de préserver une cohérence conceptuelle subtile au sein du projet, tout en reconnaissant les nuances culturelles inhérentes à ces expressions universelles si ça te va ?